java-samples/corda5-obligation-cordapp/README.md

```diff
@@ -87,3 +87,16 @@ We will now take Bob's shorthash and enter the following request Body.
 And as for the result, you need to go to the Get API again and enter the short hash and client request ID.
 
 Thus, we have concluded a full run through of the obligation app. 
+
+### App Diagrams 
+Below are the diagram of this app which are useful for the visual understanding. 
+
+#### Static Diagram
+
+![img.png](img.png)
+
+#### Dynamic Diagram
+
+![img_1.png](img_1.png)
+
+
```

Comment:
were you intending to create a diagram for obligation sample?

i think this was an accident?